core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/UniqueIDJDBCUserStoreManager.java

```diff
@@ -352,11 +352,17 @@ public List<User> doGetUserListOfRoleWithID(String roleName, String filter) thro
     }
 
     @Override
-    public int doGetUserCountOfRole(String roleName, String filter)
-                throws UserStoreException {
+    public int doGetUserCountOfRole(String roleName) throws UserStoreException {
 
-            RoleContext roleContext = createRoleContext(roleName);
-            return getUserCountByRole(roleContext, filter);
+        throw new UserStoreException("Operation is not supported.");
+    }
+
+    @Override
+    public int doGetUserCountOfRoleWithID(String roleName)
+            throws UserStoreException {
```

Comment:
can't this be in a single line?

Reply:
Addressed with https://github.com/wso2/carbon-kernel/pull/4003